lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.h

```diff
@@ -186,7 +186,7 @@ class ProxyRenderDelegate : public MHWRender::MPxSubSceneOverride
     std::unique_ptr<UsdImagingDelegate> _sceneDelegate; //!< USD scene delegate
 
     bool                    _isPopulated{ false };      //!< If false, scene delegate wasn't populated yet within render index
-    bool                    _selectionChanged{ false }; //!< Whether there is any selection change or not
+    bool                    _selectionChanged{ true };  //!< Whether there is any selection change or not
```

Comment:
At the frame creating a proxy shape, we need to update the Hydra selection.

Reply:
Because the script to create the stage will select the proxy shape, this change means the new stage will be highlighted, which is what we want (MAYA-102103).